packages/lib/modules/pool/actions/add-liquidity/form/useMaximumInputs.tsx

```diff
@@ -40,6 +40,7 @@ export function useMaximumInputs() {
       .map(balance => ({
         humanAmount: balance.formatted as HumanAmount,
         tokenAddress: balance.address as Address,
+        symbol: 'TBD', // TODO: finish with Groninge
```

Comment:
If this isn't changed it blocks the PR being merged?

Reply:
this is the last missing issue. I will try to figure it out without Groninge tomorrow (he's quite busy)

Reply:
Removed `useMaximumInputs` as it is not used anymore.